sklearn/datasets/tests/test_openml.py

```diff
@@ -147,32 +149,64 @@ def _monkey_patch_webbased_functions(context, data_id, gziped_files):
         path_suffix = '.gz'
         read_fn = gzip.open
 
+    class MockHTTPResponse():
```

Comment:
`MockHTTPResponse(object)` to use new-style classes on Py2, not that it matters much..